### ISSUE TITLE
`credentials`: `include` does not work

### ISSUE THREAD

**Issue by MinJieLiu** [apollographql/apollo-client#3389]
Browser does not carry cookies.

```js
import ApolloClient from 'apollo-boost';

const client = new ApolloClient({
  uri: 'https://nx9zvp49q7.lp.gql.zone/graphql',
  fetchOptions: {
    credentials: 'include',
  },
});
```

**Comment by hwillson** [comment]
Hi @MinJieLiu - can you supply a small runnable reproduction that demonstrates this is an issue?

**Comment by MinJieLiu** [comment]
https://github.com/apollographql/apollo-client/pull/3387

**Comment by hwillson** [comment]
Fixed in https://github.com/apollographql/apollo-client/pull/3098 and available via `apollo-client` >= 2.3.3.
